java-api/src/main/java/com/sap/cloud/security/token/SecurityContext.java

```diff
@@ -67,4 +105,12 @@ public static void clearToken() {
 		}
 	}
 
+	/**
+	 * Clears the current token and certificate from thread wide storage.
+	 */
+	public static void clear() {
+		clearCertificate();
+		clearToken();
```

Comment:
now you are calling a deprecated method...

Reply:
is there a need to clear the securitycontext i.e. separately for token and for certificate? If not then with the next major release this method needs to become private, if yes then i can de-depracete it.